app/uk/gov/hmrc/plasticpackagingtax/registration/config/AppConfig.scala

```diff
@@ -90,6 +90,9 @@ class AppConfig @Inject() (config: Configuration, val servicesConfig: ServicesCo
   lazy val soleTraderHost: String =
     servicesConfig.baseUrl("sole-trader-identification-frontend")
 
+  lazy val soleTraderJourneyInitUrl =
```

Comment:
We have the journey url below, is it not the same

Reply:
No, it's been changed slightly so that they are subtly different.